symfony/plugins/orangehrmTimePlugin/TimeSheetPeriod/DailyTimesheetPeriod.php

```diff
@@ -17,6 +17,6 @@
  * Boston, MA  02110-1301, USA
  */
 class DailyTimesheetPeriod extends TimesheetPeriod {
```

Comment:
will remove this class. since it don't have implementation

Reply:
done